iocage_lib/create_utils.py

```diff
@@ -0,0 +1,31 @@
+import os
+import shutil
+
+import iocage_lib.ioc_fstab
+
+
+def strip_jail_for_base_jail(config: dict, release: str, iocroot: str, jail_uuid: str) -> dict:
+    basedirs = [
+        'bin', 'boot', 'lib', 'libexec', 'rescue', 'sbin', 'usr/bin', 'usr/include', 'usr/lib',
+        'usr/libexec', 'usr/sbin', 'usr/share', 'usr/libdata', 'usr/lib32'
+    ]
+
+    if '-STABLE' in release:
+        # HardenedBSD does not have this.
+        basedirs.remove('usr/lib32')
+
+    for base_dir in basedirs:
+        source = os.path.join(
+            iocroot, 'templates' if all(k not in release for k in ('-RELEASE', '-STABLE')) else 'releases',
+            release, 'root', base_dir
+        )
+        destination = os.path.join(iocroot, 'jails', jail_uuid, 'root', base_dir)
+
+        # This reduces the REFER of the basejail.
+        shutil.rmtree(destination, ignore_errors=True)
```

Comment:
Why the change from rm proc to shutil?

Reply:
Seemed cleaner to remove an extra subprocess call ? In practice didn't seem to introduce a performance regression ?
We can have it if you think it would still be nice/better :)

Reply:
One less chance of regression :P

Reply:
Updated ;)